apps/zero-api/src/facilities/dto/facility.dto.ts

```diff
@@ -1,15 +1,82 @@
-import { ApiProperty } from '@nestjs/swagger';
+import { ApiProperty, ApiPropertyOptional } from '@nestjs/swagger';
 import { Exclude } from 'class-transformer';
+import {
+  DeviceOwnershipEnum,
+  DeviceRegistryEnum,
+  FacilityFinancialSupportTypeEnum,
+  RenevableEnergySourceEnum
+} from '@prisma/client';
+import { FileMetadataDto } from '../../files/dto/file-metadata.dto';
 
 export class FacilityDto {
-  @ApiProperty({ example: 1 })
-  id: number;
+  @ApiProperty({ example: '29974dc9-3582-4a09-8216-24c54b5419b1' })
+  id: string;
+
+  @ApiProperty({ example: 'A company name' })
+  companyName: string;
```

Comment:
Why don't we have type validation here ?

Reply:
It is used to create a response object. Do we need type validations on response objects?